pkg/gohai/filesystem/filesystem_nix_test.go

```diff
@@ -117,7 +117,6 @@ func TestNixFSTypeFiltering(t *testing.T) {
 			mounts, err := getFileSystemInfoWithMounts(inputMounts, mockFSSizeKB, getMockFSDev())
 			require.NoError(t, err)
 
-			require.Equal(t, len(expectedMounts), len(mounts))
```

Comment:
Why did we removed this assertion?

Reply:
It just makes the error less clear, the very next line is `assert.ElementsMatch(t, mounts, expectedMounts)`, which will fail if the length doesn't match, but it will do so by printing both slices, which is better to debug.